taskt.Studio/Commands/Image Commands/CaptureImageCommand.cs

```diff
@@ -0,0 +1,81 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Drawing;
+using System.Windows.Forms;
+using System.Xml.Serialization;
+using taskt.Core.Attributes.ClassAttributes;
+using taskt.Core.Attributes.PropertyAttributes;
+using taskt.Core.Command;
+using taskt.Core.Common;
+using taskt.Core.Enums;
+using taskt.Core.Infrastructure;
+using taskt.Engine;
+using taskt.UI.CustomControls;
+using taskt.UI.CustomControls.CustomUIControls;
+
+namespace taskt.Commands
+{
+    [Serializable]
+    [Group("Image Commands")]
+    [Description("This command captures an image on screen and stores it as a Bitmap variable.")]
+    public class CaptureImageCommand : ScriptCommand
+    {
+        [XmlAttribute]
+        [PropertyDescription("Capture Search Image")]
+        [InputSpecification("Use the tool to capture an image that will be located on screen during execution.")]
+        [SampleUsage("")]
+        [Remarks("Images with larger color variance will be found more quickly than those with a lot of white space. \n" +
+                 "For images that are primarily white space, tagging color to the top-left corner of the image and setting \n" +
+                 "the relative click position will produce faster results.")]
+        [PropertyUIHelper(UIAdditionalHelperType.ShowImageRecogitionHelper)]
```

Comment:
I think `ShowImageRecogitionHelper` should be renamed to `ShowImageCaptureHelper`

Reply:
done